docs/install/operator/configuring-eventing-cr.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Configuring the Eventing Operator Custom Resource"
+weight: 60
+type: "docs"
+aliases:
+- /docs/operator/configuring-eventing-cr/
+---
+
+The Knative Eventing operator can be configured with these options:
+
+- [Private repository and private secret](#private-repository-and-private-secrets)
+- [Configuring default broker class](#configuring-default-broker-class)
+
+__NOTE:__ Kubernetes spec level policies cannot be configured using the Knative operators.
```

Comment:
I'm not sure what this comment is supposed to mean. Are you trying to say that parts of the Eventing system components which aren't exposed by the operator can't be updated by end-users?

Reply:
I think I should remove it for confusion. It did not mean anything informative.